### DISCUSSION TITLE
How to use 3' rule and describe reference sequence on NC_(NM_) descriptions

### DISCUSSION THREAD

**Discussion by ahwagner** [HGVSnomenclature/hgvs-nomenclature#163] [Q&A]
When representing an intronic, ambiguous indel on a negative-strand-aligned NM transcript, the `c.` coordinates are with respect to the cDNA represented by the NM_ sequence, and the offsets are with respect to the genomic coordinates as they align to the transcript. When describing intronic variants using an `NC_(NM_):c.` syntax in this situation, I have the following questions:

1. Are the reference and alternate bases described with respect to the positive or negative orientation of the NC_ reference?
2. Does the 3'-rule apply to the orientation of the NM_ reference (5'-shifted WRT to NC_ reference) or the NC_ reference (5'-shifted WRT to NM_ reference)?
3. Does the 3'-rule apply consistently to intronic and exonic variants when using the `NC_(NM_):c`. syntax for negative-strand-aligned NM references?

**Comment by jfjlaros** [comment]
I would like to start with a few remarks:

> the c. coordinates are with respect to the cDNA represented by the NM_ sequence

Only when there is a 1-to-1 correspondence between the `NM` and the annotation of the chromosome. If not, the `c.` coordinates may refer to the cDNA sequence as annotated on the chromosomal reference sequence, or to the sequence in the `NM`. Which approach is "correct" is still under debate.

> the offsets are with respect to the genomic coordinates as they align to the transcript

The offsets are relative to the cDNA coordinates as annotated on the chromosomal reference sequence. As far as I know, there is no debate about this.

---

To answer your questions:

1. The negative.
2. The former, the `g.` and `c.` descriptions are shifted in opposite directions.
3. Yes.

**Reply by ahwagner** [reply]
Thanks @jfjlaros. These questions were raised because there were some statements in the nomenclature that I found confusing or ambiguous about how to handle intronic, coding DNA reference variants:

From [Numbering (introns)](https://hgvs-nomenclature.org/stable/background/numbering/?h=intronic#discussion):
> a coding DNA reference sequence does not contain intron or 5' and 3' gene flanking sequences, and can therefore not be used as a reference to describe variants in these regions (see [Reference Sequences](https://hgvs-nomenclature.org/stable/background/refseq/#DNAc)) Correct descriptions refer to a genomic reference sequence like `LRG_199t1:c.357+1G>A`, `NC_000023.10(NM_004006.2):c.357+1G>A`, or `NG_012232.1(NM_004006.2):c.357+1G>A`.

I think this language should be clarified; if the answer to 1) is the negative, then the NC_ reference is not the reference sequence; the complement of the NC_ is the reference. Not the reverse complement, though, since the +/- offsets are with respect to the alignment of the cDNA sequence with the reference forward strand coordinates.

From [Reference Sequences](https://hgvs-nomenclature.org/stable/background/refseq/#DNAc):
> when a gene is located on the minus strand, the location of a variant nucleotide may differ when described based on a genomic or a coding DNA reference sequence. Applying the 3'rule, `NC_000023.10:g.32361300del` describes the deletion of a T from a mono-nucleotide stretch in the DMD gene. On the opposite strand, `NC_000023.10:g.32361300` links to nucleotide `NM_004006.1:c.5690`. However, applying the 3'rule, based on a coding DNA reference sequence, this variant is described as `NM_004006.1:c.5697del` (linking to `NC_000023.10:g.32361293`). See also [different genomic (g.) and coding DNA (c.) descriptions](https://hgvs-nomenclature.org/stable/recommendations/DNA/repeated/).

This paragraph provides some useful information about how this applies in the exonic space, but not intronic. I think we should add a description of how this works in the intronic space. It might also be helpful to be clear that that the `T` is on the _negative strand_ (in the above example it is implied that the `T` is the nucleotide at `NC_000023.10:g.32361300del`, when in fact the forward-orientation sequence at that position is `A`.

From [Repeated Sequences example _differing genomic (g.) and coding DNA (c.) descriptions_](https://hgvs-nomenclature.org/stable/recommendations/DNA/repeated/#examples):
> NC_000001.11:g.57367047_57367121ATAAA[15] and NM_021080.3:c.-136-75952_-136-75878ATTTT[15] describe the same repeat allele in intron 3 of the DAB1 gene.
NOTE: based on the 3' rule and the transcriptional orientation of the gene (minus strand), the description of the repeat units differs.

Here, we see an example of the repeat unit syntax, and in the intronic space, making this the closest and most relevant text for answering my question. However, in this example there is a cDNA (NM_) sequence used as a reference, no NC_, seemingly in contradiction to the above guidance about not using a coding DNA reference sequence. I think this should be corrected for consistency.

I would like to propose that we create a stand-alone page that consolidates and aligns these recommendations to address these issues in an `NC_(NM_):c.` summary page, ideally with illustration. It may also be helpful to address #4 on this page.

**Reply by ahwagner** [reply]
thread continues at https://github.com/HGVSnomenclature/hgvs-nomenclature/discussions/163#discussioncomment-8918221

**Comment by jfjlaros** [comment]
Replying to https://github.com/HGVSnomenclature/hgvs-nomenclature/discussions/163#discussioncomment-8909216:
> I think this language should be clarified

Agreed. For starters, I think we should call an `NM` an RNA or transcript reference sequence. According to the GenBank annotation, the molecular type is "mRNA", the fact that the DNA alphabet is used is probably because a cDNA synthesis step was used in the lab or for other historical reasons.

Whether or not a different chromosomal reference sequence is used is debatable. We could say that the reverse complement sequence is implied by the forward sequence (which was chosen arbitrarily anyway). It would be good to at least mention the existence of transcripts on the reverse complement strand.

Perhaps we should make a separate figure like [this](https://hgvs-nomenclature--156.org.readthedocs.build/156/background/numbering/#figure) for transcripts residing on the reverse complement strand.

> the +/- offsets are with respect to the alignment of the cDNA sequence with the reference forward strand coordinates

The offsets are with respect to the `c.` coordinates that indicate the start or the end of an exon. No `g.` coordinates (forward or reverse complement) are used.

> It might also be helpful to be clear that that the `T` is on the negative strand (in the above example it is implied that the `T` is the nucleotide at `NC_000023.10:g.32361300del`, when in fact the forward-orientation sequence at that position is `A`.

Perhaps we should change the example in such a way that the nucleotides are mentioned (e.g., use an insertion instead of a deletion).  Maybe [`NC_000023.11(NM_000109.4):c.11_12insAG`](https://mutalyzer.nl/normalizer/NC_000023.11(NM_000109.4):c.11_12insAG) would be an interesting candidate for an exonic example and [`NC_000023.11(NM_000109.4):c.8-100_8-99insAG`](https://mutalyzer.nl/normalizer/NC_000023.11(NM_000109.4):c.8-100_8-99insAG) for an intronic one.

> I think this should be corrected for consistency.

Agreed, the second description is invalid.

> I would like to propose that we create a stand-alone page that consolidates and aligns these recommendations to address these issues in an `NC_(NM_):c.` summary page, ideally with illustration. It may also be helpful to address #4 on this page.

Maybe we should have a section about reference sequences and their intended use before we introduce the different coordinate systems. Perhaps this could replace the current "Numbering" page.
